scipy/optimize/_linprog.py

```diff
@@ -135,742 +141,1009 @@ def linprog_terse_callback(xk, **kwargs):
-
-        "tableau" : The current Simplex algorithm tableau
-        "nit" : The current iteration.
-        "pivot" : The pivot (row, column) used for the next iteration.
-        "phase" : Whether the algorithm is in Phase 1 or Phase 2.
-        "bv" : A structured array containing a string representation of each
-               basic variable and its current value.
-
-    Options
-    -------
-    maxiter : int
-       The maximum number of iterations to perform.
-    disp : bool
-        If True, print exit status message to sys.stdout
-    tol : float
-        The tolerance which determines when a solution is "close enough" to
-        zero in Phase 1 to be considered a basic feasible solution or close
-        enough to positive to serve as an optimal solution.
-    bland : bool
-        If True, use Bland's anti-cycling rule [3] to choose pivots to
-        prevent cycling.  If False, choose pivots which should lead to a
-        converged solution more quickly.  The latter method is subject to
-        cycling (non-convergence) in rare instances.
+        (row) in ``A_eq``. Unnecessary elements have been removed.
+    bounds : sequence of tuples
+        ``(min, max)`` pairs for each element in ``x``, defining
+        the bounds on that parameter. Use None for each of ``min`` or
+        ``max`` when there is no bound in that direction. Bounds have been
+        tightened where possible.
+    undo: list of tuples
+        (`index`, `value`) pairs that record the original index and fixed value
+        for each variable removed from the problem
 
     Returns
     -------
-    A `scipy.optimize.OptimizeResult` consisting of the following fields:
+    A : 2-D array
+        2-D array which, when matrix-multiplied by x, gives the values of the
+        equality constraints at x (for standard form problem).
+    b : 1-D array
+        1-D array of values representing the RHS of each equality constraint
+        (row) in A (for standard form problem).
+    c : 1-D array
+        Coefficients of the linear objective function to be minimized (for
+        standard form problem).
+    c0 : float
+        Constant term in objective function due to fixed (and eliminated)
+        variables.
 
-        x : ndarray
-            The independent variable vector which optimizes the linear
-            programming problem.
-        fun : float
-            Value of the objective function.
-        slack : ndarray
-            The values of the slack variables.  Each slack variable corresponds
-            to an inequality constraint.  If the slack is zero, then the
-            corresponding constraint is active.
-        success : bool
-            Returns True if the algorithm succeeded in finding an optimal
-            solution.
-        status : int
-            An integer representing the exit status of the optimization::
+    References
+    ----------
+    .. [6] Bertsimas, Dimitris, and J. Tsitsiklis. "Introduction to linear
```

Comment:
Why the references are numbered as such? They are local to the functions themselves.